build.ps1

```diff
@@ -1,68 +1,177 @@
-$ErrorActionPreference = "Stop"
-
-function DownloadWithRetry([string] $url, [string] $downloadLocation, [int] $retries)
-{
-    while($true)
-    {
-        try
-        {
-            Invoke-WebRequest $url -OutFile $downloadLocation
-            break
-        }
-        catch
-        {
-            $exceptionMessage = $_.Exception.Message
-            Write-Host "Failed to download '$url': $exceptionMessage"
-            if ($retries -gt 0) {
-                $retries--
-                Write-Host "Waiting 10 seconds before retrying. Retries left: $retries"
-                Start-Sleep -Seconds 10
+#!/usr/bin/env powershell
+#requires -version 4
+
+<#
```

Comment:
I assume these structured comments end up used in Intelisense or error messages under the correct conditions?